architecture/Threatmodel.md

```diff
@@ -0,0 +1,128 @@
+2. Misbruik van de app door de makers / beheerders van de app en de gezondheids authoriteiten. Dat is het gebruiken van de app voor andere toepassingen dan waar de gebruikers toestemming hebben gegeven en wettelijk is toegestaan. 
+Voorbeeld: De  GGD gaat in de toekomst de corona app ook gebruiken voor andere besmettelijke zieken zoals griep. Terwijl dit niet is waar de gebruikers toestemming voor hebben gegeven. 
+Voorbeeld: Terwijl de app makers claimen het veilige google/apple protocol te gebruiken wordt in werkelijkheid een ander protocol gebruikt dat niet veilig is.  
+Open source kan een gedeelte van deze zorg wegnemen maar als men wilt controleren of de gebruikte source code overeenkomt met de gepubliceerde vereist het gebruik van reproducible builds. 
+3. Gebruik van de app voor het traceren van niet zijnde COVID-19 patienten door andere overige overheid en niet overheid instanties.  
+Voorbeeld:
+
+De politie gebruikt de corona app om een alibi te controleren. De andere persoon waarbij de verdachte claimt te zijn geweest ten tijde van het misdrijf, wordt gevraagd zich als corona besmet op te geven. Als de verdachte dan geen 'corona alert' haalt dit zijn alibi onderuit. 
+Voorbeeld:  Een geheime dienst ontdekt een zwakheid in het google/apple protocol, houden dit geheim en gebruikt dit om spionnen te traceren maar de technologie die dit kan lekt uit en wordt daarna op grote schaal door prive detectives gebruikt, met een groot schandaal als gevolg.   
+## 1. Contact informatie bepalen Risico’s 
+Hiervoor wordt verwezen naar het document Crypto Raamwerk in https://github.com/minvws/nl-covid19-notification-app-coordination/tree/master/architecture
+Een kleine opmerking valt wel te maken over de test optie van het GAEN protocol. Deze is noodzakelijk om een snelle feedback te krijgen tijdens de test van de programmatuur, maar is niet wenselijk tijdens productie. De vraag is hoe toegang tot een systeemtest in de App wordt geregeld.
+## 2. Informatie over besmette contacten verkrijgen Risco's 
+De app moet regelmatig informatie van een centrale server ophalen met daarin de Temporary Exposure keys van besmette personen. 
+### Spoofing: 
+Niet van toepassing. 
+### Tampering 
+Aangenomen wordt dat alleen een proces op de server(s) de nieuwe bestanden met daarin Temporary Exposure Keys kan generen. Gebruikers toegang tot deze bestanden behalve voor backup doeleinden dient te worden vermeden. 
+### Non Repudiation 
+Niet van Toepassing 
+###  Information Disclosure
+Hierbij kunnen andere clients dan de officiele app ook de Temporary exposure keys downloaden.  Om dit te vermijden dienen er maatregelen genomen worden zodat de informatie alleen beschikbaar is voor de app. 
```

Comment:
Dit is in principe gewoon publieke informatie, dus ik vraag me af hoe dat als problematisch beschouwd zou moeten worden.

Reply:
Ik zou de Temporary Exposure Keys van besmette personen niet als publieke informatie willen classificeren. Het is versleuteld en daarom zijn de risico's veel minder. Ik zie het als een versleutelde versie van confidentiele informatie. Er zijn aanvallen denkbaar (bijvoorbeeld door malware op de telefoon die toch toegang heeft tot de Temporary exposure keys). Daarnaast is het toegang beperken tot de temporary exposure keys een methode om alternatieve clients niet te laten functioneren.

Reply:
Het is informatie die ik kan opvragen als app gebruiker, of als onafhankelijke onderzoeker. Nergens in het design staat dat deze informatie afgeschermd zou moeten worden, dus dat bedoelde ik met publieke informatie. Naast eventueel een DDoS bescherming, is het 'afschermen' van alternatieve clients onzinning. (Immers, wat is het risico? Ik kan deze informatie ook gewoon aftappen van de echte app)